spec/models/form_spec.rb

```diff
@@ -14,12 +14,14 @@
     end
   end
 
-  describe "made_live_form" do
-    let(:made_live_form) { create :made_live_form }
-    let(:form) { made_live_form.form }
+  describe ".destroy" do
+    context "when a form was made live" do
```

Comment:
What's the value of the extra `context` here?

Reply:
Because I changed the `describe` of the spec as it's about when the destroy method is called, adding the context that the state of the form is one that has a `made_live_form` child.